Source/NSubstitute/SubstituteExtensions.cs

```diff
@@ -222,7 +294,25 @@ internal static ConfiguredCall Returns<T>(MatchArgs matchArgs, Func<CallInfo, T>
         public static IEnumerable<ICall> ReceivedCalls<T>(this T substitute) where T : class
         {
             return GetRouterForSubstitute(substitute).ReceivedCalls();
-        }       
+        }
+
+#if NET4 || NET45
```

Comment:
That was a consideration, however they'd either become static calls or new extensions methods introducing new functionality to the entire codebase that, for the time being, is only used in this one class.

It would also be a NET4/NET45 only class which I felt introduces only more files to manage.

What are your thoughts? Can you think of an alterntive to a TaskExtensions / TaskHelper (wouldn't call it that but you get the gist) class?